src/extractor/scripting_environment_lua.cpp

```diff
@@ -276,6 +336,61 @@ void Sol2ScriptingEnvironment::InitContext(LuaScriptingContext &context)
         "version",
         &osmium::Way::version);
 
+    struct RelationMemberWrap
+    {
+        explicit RelationMemberWrap(const osmium::RelationMember &member) { init(member); }
+
+        RelationMemberWrap() : item_type(osmium::item_type::undefined) {}
+
+        void init(const osmium::RelationMember &member)
+        {
+            role = member.role();
+            item_type = member.type();
+            id = member.ref();
+        }
+
+        const char *GetRole() const { return role.c_str(); }
+        osmium::item_type GetItemType() const { return item_type; }
+        osmium::object_id_type GetId() const { return id; }
+
+        util::OsmIDTyped ref() const { return util::OsmIDTyped(id, std::uint8_t(item_type)); }
```

Comment:
This is the other side to my comment on `OsmIDTyped` - I think we should be very explicit about what values we store here.  Something like:

```
switch(item_type) {
  case osmium::item_type.node:
      return util::OsmIDTyped(id, osrm::extractor::OSMIDType.NODE);
  case osmium::item_type.way:
      return util::OsmIDTyped(id, osrm::extractor::OSMIDType.WAY);
  default:
     throw util::exception("some error message");
}
```

Reply:
Will do our enum with conversion